lib/daru/dataframe.rb

```diff
@@ -925,8 +926,9 @@ def all? axis=:vector, &block
     #
     # @param [Fixnum] quantity (10) The number of elements to display from the top.
     def head quantity=10
-      return Daru::DataFrame.new [self.row[0].to_a].transpose,
-        index: [@index.at(0)], order: @vectors if quantity == 1
+      return Daru::DataFrame.rows [row[0]],
+        index: [@index.at(0)],
+        order: @vectors if quantity == 1
       self[0..(quantity-1), :row]
```

Comment:
The modifying `if` on multiline statement looks ugly (even if it makes rubocop happy).
It hink, something like this should look better (and better state the logic):
```ruby
if quantity == 1
  Daru::DataFrame.rows [row[0]],
    index: [@index.at(0)],
    order: @vectors
else
  self[0..(quantity-1), :row]
end
```